### PR TITLE
Report pull metrics to Sentry

### PR DESCRIPTION
**Pull request by fkorotkov**

### CODE REVIEW

File: Sources/tart/Root.swift [hunk 1]
```diff
@@ -39,6 +39,14 @@ struct Root: AsyncParsableCommand {
       SentrySDK.start { options in
         options.dsn = dsn
         options.releaseName = CI.release
+        options.tracesSampleRate = 1.0
+
+        // By default only 5XX are captured
+        // Let's capture everything but 401 (unauthorized)
+        options.failedRequestStatusCodes = [
```
Comment (edigaryev) [inline Sources/tart/Root.swift:46]: Should we also set `enableCaptureFailedRequests` for this to work?

Comment (fkorotkov) [reply Sources/tart/Root.swift:46]: I think it's on by default but can certainly explicitly add it. 👌

File: Sources/tart/Commands/Prune.swift [hunk 2]
```diff
@@ -103,5 +104,7 @@ struct Prune: AsyncParsableCommand {
     }
 
     puppy.info("reclaimed \(cacheReclaimedBytes) bytes")
+
+    SentrySDK.span?.setMeasurement(name: "gc_disk_reclaimed", value: cacheReclaimedBytes as NSNumber, unit: MeasurementUnitInformation.byte);
```
Comment (edigaryev) [inline Sources/tart/Commands/Prune.swift:108]: I think you need to [bind the transaction to scope](https://docs.sentry.io/platforms/apple/performance/instrumentation/custom-instrumentation/#create-transaction-bound-to-the-current-scope) the first, in order for the `SentrySDK.span` to work.

Comment (fkorotkov) [reply Sources/tart/Commands/Prune.swift:108]: My bad. I saw the documentation and though it's binded by default. 😪 Good catch!

**Review by edigaryev — DISMISSED** [review]

**Review by edigaryev — DISMISSED** [review]

**Pull request merged**